src/dbnode/digest/writer_test.go

```diff
@@ -136,3 +118,40 @@ func TestFdWithDigestWriteDigestsSuccess(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expected, b)
 }
+
+func TestCloseAll(t *testing.T) {
+	writer, fd, _ := createTestFdWithDigestContentsWriter(t)
+	defer os.Remove(fd.Name()) // nolint:errcheck
```

Comment:
nit; worth asserting no errors on these? Can probably wrap these in a function and use it for all of these nolints